Sources/Help.php

```diff
@@ -109,20 +109,13 @@ function ShowAdminHelp()
 	// Allow mods to load their own language file here
 	call_integration_hook('integrate_helpadmin');
 
-	// Set the page title to something relevant.
-	$context['page_title'] = $context['forum_name'] . ' - ' . $txt['help'];
-
-	// Don't show any template layers, just the popup sub template.
-	$context['template_layers'] = array();
-	$context['sub_template'] = 'popup';
-
 	// What help string should be used?
 	if (isset($helptxt[$_GET['help']]))
 		$context['help_text'] = $helptxt[$_GET['help']];
 	elseif (isset($txt[$_GET['help']]))
 		$context['help_text'] = $txt[$_GET['help']];
 	else
-		$context['help_text'] = $_GET['help'];
+		fatal_lang_error('not_found', false, array(), 404);
```

Comment:
This might make the popup look funky. I'll need to test it to confirm.

Reply:
It just makes the popup show the string "error" in its content area.

Reply:
I guess that could work

Reply:
Yeah. The main concern is people accessing the URL directly rather than via the popup.